main.sh

```diff
@@ -39,22 +39,22 @@ awk '/EnsCodingFull/' hg19.coding_subset.bed > hg19.coding_subset.EnsCodingFull.
 awk '$4 ~ /EnsCodingFull/' hg19.coding_subset.bed > hg19.coding_subset.EnsCodingFull.bed
 
 
-
-
-
-
 ## Task8: compare the 2 files: hg19.coding_subset.bed and hg19.coding_subset.EnsCodingFull.bed to see which lines have removed.
-
+grep -F -x -v -f hg19.coding_subset.EnsCodingFull.bed hg19.coding_subset.bed > hg19.coding_subset.diff.bed
+#to get file from all other lines not in EnsCodingFull file
+#can use wc -l to determine number of lines not in EnsCodingFull file
 
 ## Task9: Cut the last 2 lines from the file "hg19.coding_subset.bed". Save in "hg19.coding_subset.truncated.bed"
-
+tail -n 2 hg19.coding_subset.bed > hg19.coding_subset.truncated.bed
```

Comment:
We need to remove the last 2 lines. This means that the output file will be missing only these 2 lines.